Update offboarding instructions (#1704)
nodejs/TSC: ONBOARDING.md

## Offboarding of voting members

* Remove them from the following GitHub orgs unless they are members for a
  reason other than TSC membership:
  * [`@nodejs-private`](https://github.com/orgs/nodejs-private/people)
  * [`@nodejs`](https://github.com/orgs/nodejs/people)

>   * [`@pkgjs`](https://github.com/orgs/pkgjs/people)

Author: ```suggestion
  * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on
     [ADMINISTRATIVE-MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/ADMINISTRATIVE-MEMBERS.md))
```

* Change the member's role from `Owner` to `Member`
  (unless they have `Owner` role for a reason other than TSC membership)
  in the following GitHub orgs:
  * [`@nodejs-private`](https://github.com/orgs/nodejs-private/people)
  * [`@nodejs`](https://github.com/orgs/nodejs/people)

>   * [`@pkgjs`](https://github.com/orgs/pkgjs/people)

Author: ```suggestion
  * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on
     [ADMINISTRATIVE-MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/ADMINISTRATIVE-MEMBERS.md))
```

## Offboarding of voting members

* Remove them from the following GitHub orgs unless they are members for a
  reason other than TSC membership:
  * [`@nodejs-private`](https://github.com/orgs/nodejs-private/people)
  * [`@nodejs`](https://github.com/orgs/nodejs/people)

>   * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on
>     [ADMINISTRATIVE-MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/ADMINISTRATIVE-MEMBERS.md))

Reviewer: They could also be in members, even if they are not administrative members.
```suggestion
  * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on
    [ADMINISTRATIVE-MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/ADMINISTRATIVE-MEMBERS.md) or in [MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/MEMBERS.md))
```

Author: Right, I meant members (I don't think we need to link both lists, do we?)
```suggestion
  * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on the
    [MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/MEMBERS.md) list)
```

Reviewer: Yep, an administrative member should be a member, so yes, just the members list should be okay.

>   * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on
>      [ADMINISTRATIVE-MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/ADMINISTRATIVE-MEMBERS.md))

Author: ```suggestion
  * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on
    [ADMINISTRATIVE-MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/ADMINISTRATIVE-MEMBERS.md))
```

* Change the member's role from `Owner` to `Member`
  (unless they have `Owner` role for a reason other than TSC membership)
  in the following GitHub orgs:
  * [`@nodejs-private`](https://github.com/orgs/nodejs-private/people)
  * [`@nodejs`](https://github.com/orgs/nodejs/people)

>   * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on
>      [ADMINISTRATIVE-MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/ADMINISTRATIVE-MEMBERS.md))

Author: ```suggestion
  * [`@pkgjs`](https://github.com/orgs/pkgjs/people) (unless they're listed on
    [ADMINISTRATIVE-MEMBERS](https://github.com/nodejs/package-maintenance/blob/main/ADMINISTRATIVE-MEMBERS.md))
```
